schema.development.kf

```diff
@@ -178,61 +178,60 @@ procedure get_inserter_or_null() private view returns (name text) {
     return null;
 }
 
-// HUMAN ACTIONS
+// USER ACTIONS
 
-procedure add_human_as_inserter($id uuid, $current_public_key text) public {
-    INSERT INTO humans (id, current_public_key, inserter) VALUES ($id, $current_public_key, get_inserter());
+procedure add_user_as_inserter($id uuid, $recipient_encryption_public_key text) public {
+    INSERT INTO users (id, recipient_encryption_public_key, inserter) VALUES ($id, $recipient_encryption_public_key, get_inserter());
 }
 
-procedure update_human_pub_key_as_inserter($id uuid, $current_public_key text) public {
+procedure update_user_pub_key_as_inserter($id uuid, $recipient_encryption_public_key text) public {
```

Comment:
I'm assuming that we can't have a more descriptive name because of Kwil identifier length limitations?

Reply:
Yes. 32 bytes is the limit.